Revert-and-Regression-Policy.md

```diff
@@ -56,17 +64,20 @@ You should also begin debugging your changes to figure out what caused the regre
 
 Once the dev workflow leads decide whether to revert, fix forward, or both, you should start work on the associated actions (re-creating the PR with the bug fixed, opening a fix-forward PR to fix the bug, or both).
 
+### Developers with Revert Permissions
+
+Any Oppia developer with permission to open revert PRs on GitHub may decide, following the above policy, whether to revert the PR, fix forward, pursue both in parallel, or make one of the two allowed exceptions. They should document their reasoning and communicate on the tracking issue so the dev workflow leads can understand what happened when they consider mitigations.
+
 ### Dev Workflow Leads
 
-Once you are informed of a regression in develop, you should make sure a tracking issue is created and alert the rest of the dev workflow leads. Decide, following the above policy, whether to revert the PR, fix forward, pursue both in parallel, or make one of the two allowed exceptions. Once a decision is made, communicate it in the tracking issue and execute it (i.e. by reverting the PR or making sure the author fixes forward).
+The dev workflow leads are developers with revert permissions, so the above instructions apply to you too. Further, leads should make sure a tracking issue is created and alert the rest of the dev workflow leads. The dev workflow leads are also responsible for making sure whatever stragety we decide on to fix develop is actually executed.
```

Comment:
Bit confusing to have the switch between second/third person here (e.g. "you" in first sentence), assuming "you" and "dev workflow leads" mean the same thing.

Reply:
fixed